WDL2CWL_example/samtoolsDocker/samtoolsSort.cwl

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env cwl-runner
```

Comment:
Did https://github.com/common-workflow-library/bio-cwl-tools/blob/release/samtools/samtools_sort.cwl not work for you?